tests/regression_tests/mg_tallies/test.py

```diff
@@ -49,7 +49,7 @@ def create_library():
     mg_cross_sections_file.export_to_hdf5('2g.h5')
 
 
-class MGXSTestHarness(HashedPyAPITestHarness):
```

Comment:
Presumably you took off this for testing purposes. 

I'm always in favor of not hashing results, as it makes it more human readable. However I think every time we discuss this I have been in the minority. Therefore if you didn't intend to keep this as unhashed, let's change it back.

Reply:
Yes, I did take it off for testing purposes. Not sure where I stood before, but I am against hashed results nowadays and think we should eventually get rid of all of them. Some of the unhashed results would be really big though, which is why they're currently hashed.